__init__.py

```diff
@@ -5,18 +5,18 @@
 Weave can be used to facilitate the creation, maintenance, and access of
 coordinated, complex data storage.
 
-Weave is used to create baskets. A basket is used to store an atomic unit (i.e.
```

Comment:
The basket/pantry capitalizations here are incorrect. The uppercase is only used when talking about the Basket/Pantry class object. Here they're talking about the things those objects represent, so it should be lower case.